resources/views/components/layout/index.blade.php

```diff
@@ -0,0 +1,13 @@
+@slots(['header', 'sidebar'])
+
+<div {{ $attributes->class('text-ct-primary flex flex-wrap gap-x-28 text-sm max-md:flex-col') }}>
+    <div class="flex flex-wrap lg:flex-row flex-col flex-1">
```

Comment:
default flex behaviour is `flex-row `so you can change this to `max-lg:flex-col`